src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -102,13 +102,21 @@
                                                                       input-state)
                                                            :stack-id current-screen-id}]))
             {fiat-currency :currency}     (rf/sub [:profile/profile])
+            disabled-from-chain-ids       (rf/sub
+                                           [:wallet/wallet-send-disabled-from-chain-ids])
             {token-symbol   :symbol
              token-networks :networks}    (rf/sub [:wallet/wallet-send-token])
+            enabled-from-networks         (->> token-networks
+                                               (filter #(not (contains? (set disabled-from-chain-ids)
+                                                                        (:chain-id %))))
+                                               set)
+            enabled-from-chain-ids        (map :chain-id enabled-from-networks)
```

Comment:
in fact if `enabled-from-chain-ids` is all you need than all of this computation should be done in the sub:

Reply:
No, its not just the `enabled-from-chain-ids`, I am also passing the `enabled-from-networks` to filter the chain icons that is displayed